lib/greenhouse_io/api/job_board.rb

```diff
@@ -54,6 +54,10 @@ def get_from_job_board_api(url, options = {})
       end
     end
 
+    def post_headers
+      { 'Authorization' => 'Basic ' + Base64.encode64(api_token), 'Content-Type' => 'application/json' }
```

Comment:
Shouldn't it be the base64 of `api_token + ':'` ?  (to be a username with no password)

Reply:
I won't disagree with you - I will test when I get a chance.
Just working from the API docs https://github.com/grnhse/greenhouse-api-docs/blob/master/source/includes/job-board/_applications.md#request-headers
and it seems to work fine without.